msal/src/internal/java/com/microsoft/identity/client/MsalUtils.java

```diff
@@ -58,18 +58,21 @@
 import java.util.StringTokenizer;
 
 /**
- * Internal Util class for MSAL.
+ * Util class for MSAL.
  */
-final class MsalUtils {
+public final class MsalUtils {
```

Comment:
Unless you're opposed to it, I think I'd prefer to see a single-purpose object for end-user tweaking of settings akin to ADAL's `AuthenticationSettings` (although, I'm admittedly not crazy about the naming of this class).

As `MsalUtils` is already a rather confusing mixture of odds 'n ends methods, I think we can reasonably create a new class which holds default values and accepts user overrides of them, possibly with inner-classes to delineate settings thematically.

For instance:
`MsalSettings.Network.CONNECT_TIMEOUT_MILLIS = 15000L;`

and maybe one day....
`MsalSettings.Logging.androidLoggingEnabled= true;`

or....
`MsalSettings.Telemetry.allowPii = true`

As a concept, I'd like to see us eventually corral our settings and flags into a unified place. I'll open an issue for discussion.

Reply:
Issue opened [here](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/249).

Reply:
Agree. I have the same thought.